tests/functional/s3/test_cp_command.py

```diff
@@ -211,19 +211,6 @@ def test_no_metadata_directive_for_non_copy(self):
         self.assertEqual(self.operations_called[0][0].name, 'PutObject')
         self.assertNotIn('MetadataDirective', self.operations_called[0][1])
 
-    def test_cp_succeeds_with_mimetype_errors(self):
```

Comment:
Yeah I saw that test but it's not exactly testing the same thing.  This test also ensures that cp succeeds when we can't get the mimetype.  I still think that's still important to test in some manner given this test was written in response to a bug fix.